### PR TITLE
Add missing attributes to download

### PR DESCRIPTION
**Pull request by gtimti**
The same for gAPI https://github.com/GapIntelligence/gapi/pull/976

### CODE REVIEW

**Comment by cheezy2022** [comment]
🆒

File: lib/gap_intelligence/models/record.rb [hunk 1]
```diff
@@ -1,8 +1,5 @@
 module GapIntelligence
   class Record
-    attr_reader :id,
```
Comment (shavkat-samatov) [inline lib/gap_intelligence/models/record.rb]: wow... great catch!

File: lib/gap_intelligence/models/download.rb [hunk 2]
```diff
@@ -1,9 +1,11 @@
 module GapIntelligence
   class Download < Record
     attributes :start_at, :end_at, class: Date
+    attribute :created_at, class: Time
 
     attributes :category_names
     attribute :country_names, class: Array
+    attribute :report_types, class: Array
```
Comment (shavkat-samatov) [inline lib/gap_intelligence/models/download.rb]: totally agree with that part!

File: lib/gap_intelligence/models/download.rb [hunk 3]
```diff
@@ -1,9 +1,11 @@
 module GapIntelligence
   class Download < Record
     attributes :start_at, :end_at, class: Date
+    attribute :created_at, class: Time
```
Comment (shavkat-samatov) [inline lib/gap_intelligence/models/download.rb]: why do we need created_at, I did not see api returning created_at from downloads endpoint

Comment (shavkat-samatov) [inline lib/gap_intelligence/models/download.rb]: please disregard previous comment, i did not read PR description properly 🙈

**Comment by shavkat-samatov** [comment]
👍

**Pull request merged**